src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -89,7 +87,7 @@ const UsersFragment = ({
   const [profile] = React.useContext(Context).profile;
   const navigation = useNavigation();
   const [client] = React.useContext(Context).client;
-  const {exhangeFollower, users, updateFollowDiscoveryContext} = useDiscovery();
+  const {exchangeFollower, users, updateFollowDiscoveryContext} = useDiscovery();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a typo in the old hunk, `exhangeFollower`, which has been corrected to `exchangeFollower` in the new hunk. Good fix.
```diff
- const {exhangeFollower, users, updateFollowDiscoveryContext} = useDiscovery();
+ const {exchangeFollower, users, updateFollowDiscoveryContext} = useDiscovery();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->